src/views/LandingPage/LandingPage.js

```diff
@@ -123,7 +123,7 @@ function LandingPage(props) {
 
 			<Reveal>{timeline}</Reveal>
```

Comment:
What is the use of {timeline} here? Seems unnecessary. If you comment this out, I guess you wouldn't have to even comment out lines 64 and 65 in LandingPage.js (I guess, but make sure this guess is right :p)